pkgs/tools/networking/curl/default.nix

```diff
@@ -120,6 +118,7 @@ stdenv.mkDerivation (finalAttrs: {
       (lib.withFeature rtmpSupport "librtmp")
       (lib.withFeature rustlsSupport "rustls")
       (lib.withFeature zstdSupport "zstd")
+      (lib.withFeature pslSupport "libpsl")
```

Comment:
```suggestion
```
this gets autodetected when pslSupport is true. (added in 1e4864e29574992c4e51af7eafe8dc65f8181c3d) is there a reason to explicitly enable / disable it?

Reply:
Curl currently requires explicitly specifying that PSL support should be disabled.
Error without this line:
```
       > checking whether to use builtin CA store of SSL library... yes
       > checking for library containing psl_builtin... no
       > configure: error: libpsl was not found
```

Reply:
ah -- i was building 8.5.0.